i18n/en.json

```diff
@@ -7328,12 +7328,16 @@
   },
   {
     "id": "utils.mail.send_mail.from_address.app_error",
-    "translation": "Notification From Address setting is missing or invalid."
+    "translation": "Error setting from address"
```

Comment:
I think is clearer to use upper case for "From", to be clear about which kind of "From" are we talking about. The same for the "To" in this PR too.

Now I read that there was an error setting something from some address, or setting something to an address.

Reply:
Hmm yeah I lower-cased it because it was confusing with all the other capitalization. What do you think about capitalizing it and also putting it in quotes `"From Address"`?

Reply:
Good point. I wonder if the initial error message actually works if we just have it in quotation marks

```
"Notification From Address" setting is missing or invalid.
```

Since then it's obvious that setting is missing or invalid.

Reply:
Quoting "From Address" or "Notification From Address" works for me :)

Reply:
I'll update to add capitalization and quotes but I purposely removed the `is missing or invalid` bit because that's not always accurate